bin/bump-version.rb

```diff
@@ -49,7 +49,9 @@
 puts
 puts "git add CHANGELOG.md common/lib/dependabot/version.rb"
 puts "git commit -m 'v#{new_version}'"
-puts "git push origin main"
+puts "git push origin HEAD:v#{new_version}-release-notes"
+puts "... create PR, verify, merge"
```

Comment:
A [gh](https://github.com/cli/cli) command to open the PR would be 😎 , but I wasn't 100% what it would be (or if it was a safe scripting assumption).

Reply:
I'd dig it and I think `gh pr create` would be the command? Maybe:

```
puts "... create PR, verify, merge, for example:"
puts "gh pr create"
```